evals/elsuite/audio/eval.py

```diff
@@ -411,3 +411,27 @@ def compute_corpus_metrics(self):
         events = self.get_match_events()
         metrics["score"] = sum(e.data["score"] for e in events) / len(events)
         return metrics
+
+
+class SpokenCompare(MatchAudioTask):
+    TASK_PROMPT = "{{instruction}}\n1. {AUDIO_PLACEHOLDER}\n2. {AUDIO_PLACEHOLDER}\nOnly respond with either '1' or '2'."
```

Comment:
Just thinking out loud:

Repeating it probably doesn't hurt, but I see instructions already include "respond with 1 or 2" in some form or another.

Reply:
Yeah, that's what I had hoped, but without this typically the model would reply with a long-winded response about why it chose one or the other.